.github/workflows/preliminary_checks.yml

```diff
@@ -1,77 +0,0 @@
-# Copyright (c) Facebook, Inc. and its affiliates.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-name: Run Checks
-
-on:
-  pull_request:
-    types:
-      - opened
-      - reopened
-      - edited
-      - synchronize
-
-permissions:
-  contents: read
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.repository }}-${{ github.head_ref || github.sha }}
-  cancel-in-progress: true
-
-jobs:
-  check-matrix:
-    name: ${{ matrix.config.name }}
-    runs-on: ubuntu-latest
-    container: ghcr.io/facebookincubator/velox-dev:check
-    strategy:
-      fail-fast: false
-      matrix:
-        config:
-          - { name: "License Header",
-              command: "header-fix",
-              message: "Found missing License Header(s)",
-            }
-          - { name: "Code Format",
-              command: "format-fix",
-              message: "Found format issues"
-            }
```

Comment:
If we delete this workflow, what will run the header-fix and format-fix pieces?

Reply:
Gotcha, sorry, I forgot what we discussed offline. Can you add it into the PR description so that we remember?